### PR TITLE
Alternate elements

### PR DESCRIPTION
**Pull request by zamattiac**
If something is intentionally blank ("You haven't created any nodes" in a list of your nodes, for example), it spot checks for that instead of a complete element. This implements a dictionary instead of a list.

### CODE REVIEW

File: verifier.py [hunk 1]
```diff
@@ -25,7 +28,7 @@
 
 
 # Takes a URL and produces its relative file name.
-def get_path_from_url(self, url):
+def get_path_from_url(url):
     # Remove http://domain
```
Comment (adlius) [inline verifier.py]: This method, along with "generate_page_dictionary()", could both become private instance methods of the class.

Comment (zamattiac) [inline verifier.py]: @adlius We could. I believe you or Sadiyah or Cam gave a reason not to though..?
@sf2ne @cameronblandford

Comment (adlius) [inline verifier.py]: It could be a private utility method. Leaving them out doesn't seem very clean.

Comment (sf2ne) [inline verifier.py]: I can't think of any reason why it would be a bad idea.

**Comment by zamattiac** [comment]
Made method private to Page class.

File: verifier.py [hunk 2]
```diff
@@ -27,30 +27,12 @@
...
-# Takes a URL and produces its relative file name.
-def get_path_from_url(url):
-    # Remove http://domain
-    tail = url.replace(base_urls[0], '') + 'index.html'
-    path = MIRROR + tail
-    return path
-
-
-# Creates a dictionary with filename : URL for all the URLs found by the crawler in the API
-def generate_page_dictionary(self):
```
Comment (sf2ne) [inline verifier.py:36]: Out of curiosity, where did this method go?

Comment (zamattiac) [inline verifier.py:36]: @sf2ne It's in the Page superclass, as you asked.

Comment (sf2ne) [inline verifier.py:36]: @zamattiac Then why is generate_page_dictionary not showing up in the page superclass in the diff?

Comment (zamattiac) [inline verifier.py:36]: generate_page_dictionary was deprecated. That was before we implemented an object approach for pages.

**Pull request merged**